Tests/Builder/ListBuilderTest.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrineORMAdminBundle\Tests\Builder;
+
+use Prophecy\Argument;
+use Prophecy\Prophet;
+use Sonata\AdminBundle\Admin\AdminInterface;
+use Sonata\AdminBundle\Guesser\TypeGuesserInterface;
+use Sonata\DoctrineORMAdminBundle\Admin\FieldDescription;
+use Sonata\DoctrineORMAdminBundle\Builder\ListBuilder;
+use Sonata\DoctrineORMAdminBundle\Model\ModelManager;
+use Symfony\Component\Form\Guess\Guess;
+use Symfony\Component\Form\Guess\TypeGuess;
+
+/**
+ * @author Andrew Mor-Yaroslavtsev <andrejs@gmail.com>
+ */
+class ListBuilderTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var TypeGuesserInterface|\Prophecy\Prophecy\ObjectProphecy
+     */
+    protected $typeGuesser;
+
+    /**
+     * @var ListBuilder
+     */
+    protected $listBuilder;
+
+    /**
+     * @var AdminInterface|\Prophecy\Prophecy\ObjectProphecy
+     */
+    protected $admin;
+
+    /**
+     * @var ModelManager|\Prophecy\Prophecy\ObjectProphecy
+     */
+    protected $modelManager;
+
+    protected function setUp()
+    {
+        $prophet = new Prophet();
+
+        $this->typeGuesser = $prophet->prophesize(
+            'Sonata\AdminBundle\Guesser\TypeGuesserInterface'
```

Comment:
No need to linebreak this, is not under 80/100/120 chars right? (this applies for the other linebreaks)